src/app/components/scroll.jsx

```diff
@@ -14,7 +14,34 @@ var mockDataThumb1 = {
 
 class Scroll extends React.Component{
   render() {
-    const scrollbarStyles = {borderRadius: 5};
+    var scrollbarStyles = {borderRadius: 5};
+    var assemblyStyles = {
+      width: '100%',
+    	paddingBottom: '25%',
```

Comment:
with CSS attributes like padding, can you use this syntax:
```
padding: 0 10px 25% 10px;
```
to save a few lines?